### PR TITLE
change template format and text

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=2592098&fileview=grid

### CODE REVIEW

File: resources/views/emails/auth/reset_password_successfull.blade.php [hunk 1]
```diff
@@ -3,24 +3,14 @@
...
         <tbody>
-        <tr>
-            <td align="center" style="font-size:0px;padding:10px 25px;padding-right:25px;padding-left:25px;word-break:break-word;">
-                <div style="font-family:open Sans Helvetica, Arial, sans-serif;font-size:16px;line-height:1;text-align:center;color:#000000;">Dear <a href="#" style="text-decoration:none !important;color:black !important; cursor:default !important">{!!$user_email!!}</a>,</div>
-            </td>
-        </tr>
         <tr>
             <td align="center" style="font-size:0px;padding:10px 25px;word-break:break-word;">
-                <div style="font-family:open Sans Helvetica, Arial, sans-serif;font-size:16px;line-height:1;text-align:center;color:#000000;">You are receiving this email because you changed your password.</div>
+                <div style="font-family:open Sans Helvetica, Arial, sans-serif;font-size:16px;line-height:1;text-align:center;color:#000000;">The password was reset for FNid <a href="#" style="text-decoration:none !important;color:black !important; cursor:default !important; font-weight:bold">{!!$user_email!!}</a>.</div>
```
Comment (smarcet) [inline resources/views/emails/auth/reset_password_successfull.blade.php:8]: please dont explicitly mention **FNid** instead use {!! Config::get("app.name") !!}

**Review by smarcet — CHANGES_REQUESTED** [review]
please review comments

File: resources/views/emails/auth/reset_password_successfull.blade.php (hunk 1 above)
Comment (romanetar) [reply resources/views/emails/auth/reset_password_successfull.blade.php]: this has been fixed

**Review by smarcet — APPROVED** [review]
looks good to me

**Pull request merged**